### PR TITLE
Update bus.json

### PR DESCRIPTION
**Pull request by kjonosm**
- add wikidata & wikipedia
- delete operator-only entries
- delete duplicates (mostly abbreviations as network name)

### CODE REVIEW

**Comment by bhousel** [comment]
Like #4626 I want to wait a few days and see if the community is onboard with this kind of `network`/`network:short` remapping.  I think it's a good thing!

File: data/transit/route/bus.json [hunk 1]
```diff
@@ -4708,22 +4692,15 @@
...
-      "locationSet": {"include": ["at"]},
-      "tags": {
-        "network": "Kärnter Linien",
-        "operator": "Kärnter Linien",
-        "route": "bus"
-      }
-    },
-    {
-      "displayName": "Kärntner Linien",
+      "displayName": "Verkehrsverbund Kärnten",
```
Comment (hikemaniac) [inline data/transit/route/bus.json:4695]: Hi @kjonosm! As far as I know "Verkehrsverbund Kärnten" is the name of the legal entity behind the network, but as a ground truth there is only the name "Kärntner Linien" to be found. None of the bus tickets or bus stops bear the name "Verkehrsverbund Kärntnen", you'll only see "Kärntner Linien". Even on their webpage, you'll have to stumble upon the legal notice to find "Verkehrsverbund Kärnten".

File: data/transit/route/bus.json [hunk 2]
```diff
@@ -2586,12 +2586,14 @@
...
     {
-      "displayName": "DE_CO_BUS",
+      "displayName": "Verkehrsgemeinschaft Coburg",
       "id": "decobus-3b9544",
       "locationSet": {"include": ["de"]},
       "tags": {
-        "network": "DE_CO_BUS",
-        "operator": "DE_CO_BUS",
+        "network": "Verkehrsgemeinschaft Coburg",
+        "network": "VGC",
```
Comment (peternewman) [inline data/transit/route/bus.json:2594]: I think you mean:

```suggestion
        "network": "Verkehrsgemeinschaft Coburg",
        "network:short": "VGC",
```

Comment (kjonosm) [reply data/transit/route/bus.json:2594]: Thanks for the review! As there is no common approach on naming networks in Germany (abbreviated or not?) I decided to close this pull request in lieu of another one with names left untouched but additional wikidata, wikipedia and some minor changes.

File: data/transit/route/bus.json (hunk 1 above)
Comment (kjonosm) [reply data/transit/route/bus.json:4695]: Thanks for the review! As there is no common approach on naming networks in Germany (abbreviated or not?) I decided to close this pull request in lieu of another one with names left untouched but additional wikidata, wikipedia and some minor changes.

**Comment by kjonosm** [comment]
Thanks for the review! As there is no common approach on naming networks in Germany (abbreviated or not?) I decided to close this pull request in lieu of another one with names left untouched but additional wikidata, wikipedia and some minor changes.

**Pull request closed without merging**